tests/verify_run.cpp

```diff
@@ -122,7 +122,14 @@ int authenticate_json(const std::string &json_file, const bool compute_validatio
 	Eigen::MatrixXd sol;
 	Eigen::MatrixXd pressure;
 
-	state.solve_problem(sol, pressure);
```

Comment:
why this change?

Reply:
@zfergus

Reply:
So it will continue testing the rest of the scenes and not crash when it runs out of iterations.